### PR TITLE
Trained the usage of pseudo-classes

### PR DESCRIPTION
**Pull request by MItan9**

### CODE REVIEW

File: HW_1/styles.css [hunk 1]
```diff
@@ -1,14 +1,79 @@
...
+    color: rgb(0, 46, 67);
+}
+
+a.nav_color {
+    color: cornsilk;
+}
+
+a:hover {
+    color: rgb(89, 82, 3);
+    display: inline;
```
Comment (IvanCeban) [inline HW_1/styles.css:23]: what for are you setting "display: inline" here?

**Review by IvanCeban — APPROVED** [review]
Well done!

**Review by NedealcovaIrina — APPROVED** [review]
Good job, Tatiana

File: HW_1/styles.css (hunk 1 above)
Comment (MItan9) [reply HW_1/styles.css]: It put it in the wrong place, in this particular place it is doing nothing

**Pull request merged**